ISSUE_TEMPLATE/stable.md

```diff
@@ -0,0 +1,97 @@
+First, verify that you meet all the [prerequisites](https://github.com/coreos/fedora-coreos-streams/blob/master/release-prereqs.md)
+
+Name this issue `stable: new release on YYYY-MM-DD` with today's date. Once the pipeline spits out the new version ID, you can append it to the title e.g. `(31.20191117.3.0)`.
+
+# Pre-release
+
+## Promote testing changes
```

Comment:
can we update this to say `Promote testing changes to stable` and also update the testing.md to say `Promote testing-devel changes to testing`?

Reply:
Done!